src/sparseml/transformers/utils/helpers.py

```diff
@@ -16,10 +16,91 @@
 Helper variables and functions for integrating SparseML with huggingface/transformers
 flows
 """
+import logging
+import os
+from typing import Optional
 
-__all__ = [
-    "RECIPE_NAME",
-]
+from sparsezoo.v2.helpers import setup_model_directory
+
+
+__all__ = ["RECIPE_NAME", "get_model_directory"]
 
 
 RECIPE_NAME = "recipe.yaml"
+TRAINING_FILES = [
+    "eval_results.json",
+    "training_args.bin",
+    "vocab.txt",
+    "all_results.json",
+    "trainer_state.json",
+    "special_tokens_map.json",
+    "train_results.json",
+    RECIPE_NAME,
+    "pytorch_model.bin",
+    "config.json",
+    "tokenizer_config.json",
+    "tokenizer.json",
+]
+DEPLOYMENT_FILES = ["tokenizer.json", "config.json", "model.onnx"]
+
+
+def get_model_directory(
+    output_dir: str,
+    training_outputs_dir: str,
+    logs_path: Optional[str] = None,
+    training_files: str = TRAINING_FILES,
+    deployment_files: str = DEPLOYMENT_FILES,
+) -> None:
```

Comment:
nit: remove `None` return typing
```suggestion
):
```